denonavr/denonavr.py

```diff
@@ -175,7 +175,7 @@ async def async_setup(self) -> None:
 
             for zone_name, zone_item in self._zones.items():
                 if zone_name != self.zone:
-                    await zone_item.async_setup()
+                    await zone_item.async_setup()  # TODO: This method doesn't seem to exist in DenonAVRFoundation?
```

Comment:
The zone is of type `DenonAVR` too (see[ here](https://github.com/ol-iver/denonavr/blob/680c33d5347c50660bd82603e5568dd1df75bbce/denonavr/denonavr.py#L150-L157)). That's why it is working 😄 

However, I used the parent class (`DenonAVRFoundation`) as type because the class cannot reference itself in type annotations.
You could try this in line 88 of the same file.
```python
    _zones: Dict[str, "DenonAVR"] = attr.ib(
```